<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zDlgSettings</class>
 <widget class="QDialog" name="zDlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zBrainSystem.qrc">
    <normaloff>:/zBrainSystem/Resources/gear.png</normaloff>:/zBrainSystem/Resources/gear.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="m_tvSettings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_tabComands">
      <attribute name="title">
       <string>Команды</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="m_btnAdd">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Добавить</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="zBrainSystem.qrc">
             <normaloff>:/zBrainSystem/Resources/add.png</normaloff>:/zBrainSystem/Resources/add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnEdit">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Редактировать</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="zBrainSystem.qrc">
             <normaloff>:/zBrainSystem/Resources/96.png</normaloff>:/zBrainSystem/Resources/96.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnDelete">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Удалить</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="zBrainSystem.qrc">
             <normaloff>:/zBrainSystem/Resources/delete.png</normaloff>:/zBrainSystem/Resources/delete.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnClear">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Очистить</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="zBrainSystem.qrc">
             <normaloff>:/zBrainSystem/Resources/43096_brush.png</normaloff>:/zBrainSystem/Resources/43096_brush.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="m_tvCommands">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_TabGeneral">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_gbType">
         <property name="title">
          <string>Игра</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="m_rbBrainRing">
            <property name="text">
             <string>Брейн-ринг</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_rbSvojaIgra">
            <property name="text">
             <string>Своя игра</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="m_gbSound">
         <property name="title">
          <string>Звук</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="m_rbPcSpeacker">
            <property name="text">
             <string>Через PC Speaker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_rbSoundCard">
            <property name="text">
             <string>Через звуковую карту</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="m_gbBackground">
         <property name="title">
          <string>Подложка</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLineEdit" name="m_leBackground">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_btnBrowseBackground">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="m_gbKeys">
         <property name="title">
          <string>Кнопки</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lStart">
            <property name="text">
             <string>Старт:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QKeySequenceEdit" name="m_ksStart"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lStop">
            <property name="text">
             <string>Стоп:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QKeySequenceEdit" name="m_ksStop"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabBreanRing">
      <attribute name="title">
       <string>Брейн-ринг</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="m_lMainTimeB">
             <property name="text">
              <string>Основное время:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="m_sbBrainMainTime">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>сек.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="m_lTime1">
               <property name="text">
                <string>После 1-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime1">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="m_lTime2">
               <property name="text">
                <string>После 2-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime2">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="m_lTime3">
               <property name="text">
                <string>После 3-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime3">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="m_lTime4">
               <property name="text">
                <string>После 4-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime4">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="m_lTime5">
               <property name="text">
                <string>После 5-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime5">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="m_lTime6">
               <property name="text">
                <string>После 6-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime6">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="m_lTime7">
               <property name="text">
                <string>После 7-й ошибки:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime7">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="m_lTime">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>После &gt; 7 ошибок:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="m_sbBrainTime">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>сек.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabSvojaIgra">
      <attribute name="title">
       <string>Своя игра</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="m_lMainTimeB_2">
           <property name="text">
            <string>Основное время:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_sbSvoyMainTime">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>сек.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="m_cbFalseStart">
         <property name="text">
          <string>Включить фальстарт</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>265</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="zBrainSystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>addCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnEdit</sender>
   <signal>clicked()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>editCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>deleteCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tvCommands</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>zDlgSettings</receiver>
   <slot>editCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_tvCommands</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>zDlgSettings</receiver>
   <slot>updateButtons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnClear</sender>
   <signal>clicked()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>clearCommands()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnBrowseBackground</sender>
   <signal>clicked()</signal>
   <receiver>zDlgSettings</receiver>
   <slot>onBrowseBackground()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCommand()</slot>
  <slot>editCommand()</slot>
  <slot>deleteCommand()</slot>
  <slot>updateButtons()</slot>
  <slot>clearCommands()</slot>
  <slot>save()</slot>
  <slot>onBrowseBackground()</slot>
 </slots>
</ui>
